Add SAP reqs (#218)
SUSE/doc-unversioned: trento/adoc/trento-requirements.adoc

A production-ready {k8s}-based deployment of {trserver} requires {k8s} knowledge. The Helm chart is intended to be used by customers without in-house {k8s} expertise, or as a way to try {trento} with a minimum of effort. However, Helm chart delivers a basic deployment of the {trserver} with all the components running on a single node of the cluster.

[[sec-sap-requirements]]
=== {sap} requirements

> {trento} must run on on a HANA database to be able to discover an {sap} system. In addition, the parameter `dbs/hdb/dbname` in the DEFAULT profile of the {sap} system must be set to the database (tenant) name as its value.

Reviewer: ```suggestion
An {sap} system must run on a HANA database to be discovered by {trento} . In addition, the parameter `dbs/hdb/dbname` must be set in the DEFAULT profile of the {sap} system to the right database (tenant) name.
```
